pom.xml

```diff
@@ -522,6 +522,9 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <skipTests>true</skipTests>
+                        </configuration>
```

Comment:
I'm not keen on this. I very often run the `IT` profile with the intention of running both unit and integration tests.

Reply:
The reason for this change is we are running unit tests in all the integration tests builds on CI, which is unnecessary and increases build time. It's very easy to run both unit and integration tests on local with a slightly different command or we can come up with a different profile to run both unit and integration tests which is used only on local.
Anyways I am happy to revert the change if you feel strong about it.

Reply:
let's separate this change out from the rest of the changes here. If you could put it in a separate PR so we can discuss just this change there that'd be awesome